app/javascript/components/visualization/controls/AnnotationSelector.js

```diff
@@ -20,6 +20,11 @@ function getAnnotationOptions(annotationList, clusterName) {
     options: annotationList.annotations
       .filter(annot => annot.cluster_name === clusterName && annot.scope === 'user')
       .map(annot => annotationKeyProperties(annot))
+  }, {
+    label: 'Cannot Display',
+    options: annotationList.annotations
+      .filter(annot => annot.scope == 'invalid' && (annot.cluster_name == clusterName || !annot.cluster_name))
```

Comment:
Best to use '===' rather than '==' to be safest

Reply:
fb73f5e